src/api/app/policies/canned_response_policy.rb

```diff
@@ -1,4 +1,10 @@
 class CannedResponsePolicy < ApplicationPolicy
+  class Scope < ApplicationPolicy::Scope
```

Comment:
I mean sure we can do it with a policy scope but we could also just do

```
CannedResponsesDropdownComponent.new(User.session.canned_responses.where(decision_type: nil).order(:title))
```

in `app/views/webui/comment/_comment_field.html.haml`. Because this scope will never have an `if, else` right?

Reply:
This scope is used in two places, so this was an easier way to do it :D

Reply:
Resolved